common-packages/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
 common_packages:
-  - language-pack-fr
+  - language-pack-en
```

Comment:
this was causing an issue with `unarchive` command

see https://github.com/ansible/ansible/issues/11348